ag/ag.proto

```diff
@@ -250,6 +250,7 @@ message LabRequest {
     uint64 courseID = 1;
     uint64 submissionID = 2;
     uint64 assignmentID = 3;
+    bool groupLabs = 4;
```

Comment:
Can’t this be derived from the assignment?

Reply:
Or is this because we want to avoid a database lookup to determine whether or not it is a group lab?

Reply:
We use this request to rebuild a certain lab (here both submission and lab IDs are provided), and to fetch all the course labs (that is where we need to distinguish between individual and personal labs). 
Should we rather split it into two different request message types?

Reply:
I was think about that as an option. Still thinking. Will take another look and think if that would make sense.